static/js/filter.js

```diff
@@ -216,6 +216,7 @@ function encodeOperand(operand) {
 }
 
 function decodeOperand(encoded, operator) {
+    encoded = encoded.replace(/"/g, '');
```

Comment:
Not an essential case and current improvement is far far better than considering this case but still I am gonna run it by you thinking you might have some quick idea's about it.
So consider the case when the stream name itself has a `"`. In this case our functioning would go wrong and will cause search suggestions to disappear as we type in the `"` since we can't differentiate about whether its a query closing quote or quote from stream name.

As I said, it might not be that important but surly a thing to ponder about if there exits a simple fix.